src/Behavioral.Automation/Bindings/ListBinding.cs

```diff
@@ -57,8 +55,8 @@ public void CheckListHaveItems(IListWrapper list, string behavior, Table table)
         /// | Test value 1 |
         /// | Test value 2 |
         /// </example>
-        [Given("(.+?) (contains|does not contain) the following items:")]
-        [Then("(.+?) should (contain|not contain) the following items:")]
+        [Given("the (.+?) (contains|does not contain) the following items:")]
```

Comment:
Not sure that these are required changes

Reply:
Discussed to leave changes as is